sys/drv/rtc.c

```diff
@@ -126,11 +129,12 @@ static int rtc_attach(device_t *dev) {
   return 0;
 }
 
-static driver_t rtc_driver = {
-  .desc = "MC146818 RTC driver",
-  .size = sizeof(rtc_state_t),
-  .attach = rtc_attach,
-  .identify = bus_generic_identify,
-};
+DEVICE_DRIVER_GEN_PCI_PROBE(RTC_VENDOR_ID, RTC_DEVICE_ID, rtc)
+
+static driver_t rtc_driver = {.desc = "MC146818 RTC driver",
+                              .size = sizeof(rtc_state_t),
+                              .attach = rtc_attach,
+                              .identify = bus_generic_identify,
+                              .probe = dev_generic_probe_rtc};
```

Comment:
Why did you change the formatting here?

Reply:
I misued make format, it's corrected now.